lib/modules/header-widgets/index.js

```diff
@@ -129,6 +153,18 @@ module.exports = {
                     widget.imageId = imageId;
                     widget.formattedImageStyles = styleSchema.format(imageId, widget.imageStyles);
                 }
+                
+                if (widget.hasFixedIdeaCount) {
+                    widget.ideaCountParts = widget.fixedIdeaCount ? widget.fixedIdeaCount.substring(0, 3).split("") : [];
+                } else {
+                    widget.ideaCountParts = req.data.ideas ? req.data.ideas.length.toString().split("") : 0;
+                }
+                
+                if (widget.ideaCountParts.length < 3) {
+                    while (widget.ideaCountParts.length < 3) {
+                        widget.ideaCountParts = [0, ...widget.ideaCountParts];
+                    }
+                }
```

Comment:
I've made this code as concise as possible while keeping it readable. Is there a better / easier way to achieve this?

Reply:
Maybe you can move this to a separate method something like: `generateIdeaCountParts`